src/async_impl/client.rs

```diff
@@ -1326,6 +1330,17 @@ impl ClientBuilder {
         self
     }
 
+    // Retry options
+
+    /// Set a request retry policy.
+    ///
+    /// Default behavior is to retry protocol NACKs.
+    // XXX: accept an `impl retry::IntoPolicy` instead?
```

Comment:
This question is about accept a sealed generic instead. The reason I wonder this is that, it could allow us to _eventually_ also allow accepting a type that implements `tower::retry::Policy` directly (we don't currently since tower is less stable).

While we _could_ change to a generic later and it's mostly not breaking, it has the potential to mess with inference if any was using some local `Into<retry::Builder>` or something, I suppose.